packages/esm-ward-app/src/ward-view/ward-view.component.tsx

```diff
@@ -19,7 +20,6 @@ const WardView = () => {
   const { t } = useTranslation();
   const isBedManagementModuleInstalled = useFeatureFlag('bedmanagement-module');
 
-  //TODO:Display patients with admitted status (based on their observations) that have no beds assigned
   if (!isBedManagementModuleInstalled || isLoadingLocation) {
```

Comment:
Won't this still be an issue @mogoodrich ?  Can the code make it past this point?

Reply:
Oh, good point, will fix... it gives me an answer to another question I had as well, didn't see we had a variable to determine if the module was installed or not.